### PR TITLE
Rewrite and reformat BUILD.md

### PR DESCRIPTION
**Pull request by mpreisler**
Improved BUILD.md, it was not too bad but some of the formatting is jarring and some of the info was outdated.

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -2,144 +2,232 @@
...
+
+(optional) Install the `ninja` build system if you want to use it instead of `make` for faster builds:
+
+```
+# yum install ninja-build
+```
+
+### Downloading the source code
+
+Download and extract a tarball from the [https://github.com/OpenSCAP/scap-security-guide/releases](list of releases):
```
Comment (matejak) [inline BUILD.md]: The HTML links are done differently in adoc.

File: BUILD.md [hunk 2]
```diff
@@ -2,144 +2,232 @@
...
+```
+# change X.Y.Z for desired version
+wget https://github.com/OpenSCAP/scap-security-guide/releases/download/vX.Y.Z/scap-security-guide-X.Y.Z.tar.bz2
+tar -xvjf ./scap-security-guide-X.Y.Z.tar.bz2
+cd ./scap-security-guide-X.Y.Z/
+```
```
Comment (matejak) [inline BUILD.md]: I suggest defining `ssg_version=x.y.z` variable and then using it, to make copy-pasting easier.

**Comment by konstruktoid** [comment]
Markdown lint:
```
$ mdl BUILD.md
BUILD.md:7: MD013 Line length
BUILD.md:18: MD013 Line length
BUILD.md:23: MD013 Line length
BUILD.md:33: MD013 Line length
BUILD.md:41: MD013 Line length
BUILD.md:69: MD013 Line length
BUILD.md:197: MD013 Line length
BUILD.md:215: MD013 Line length
BUILD.md:225: MD013 Line length
BUILD.md:62: MD014 Dollar signs used before commands without showing output
BUILD.md:79: MD014 Dollar signs used before commands without showing output
BUILD.md:59: MD022 Headers should be surrounded by blank lines
BUILD.md:186: MD022 Headers should be surrounded by blank lines
BUILD.md:8: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:19: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:24: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:29: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:42: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:50: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:70: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:91: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:99: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:116: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:126: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:137: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:152: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:161: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:170: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:181: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:203: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:214: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:221: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:226: MD031 Fenced code blocks should be surrounded by blank lines
BUILD.md:231: MD031 Fenced code blocks should be surrounded by blank lines
```

**Comment by matejak** [comment]
@konstruktoid Thank you for the analyses, if @mpreisler doesn't incorporate those fixes into his PR, you are warmly welcome to do it yourself!

File: BUILD.md (hunk 1 above)
Comment (mpreisler) [reply BUILD.md]: fixed

File: BUILD.md (hunk 2 above)
Comment (mpreisler) [reply BUILD.md]: fixed

**Comment by matejak** [comment]
Nice, thank you once again! BTW the markdown syntax low-hanging fruit is still waiting to be picked up.

**Pull request merged**